cluster-autoscaler/simulator/cluster.go

```diff
@@ -113,14 +111,20 @@ type UtilizationInfo struct {
 
 // FindNodesToRemove finds nodes that can be removed. Returns also an information about good
 // rescheduling location for each of the pods.
-func FindNodesToRemove(candidates []*apiv1.Node, destinationNodes []*apiv1.Node, pods []*apiv1.Pod,
-	listers kube_util.ListerRegistry, clusterSnapshot ClusterSnapshot, predicateChecker PredicateChecker, maxCount int,
-	fastCheck bool, oldHints map[string]string, usageTracker *UsageTracker,
+func FindNodesToRemove(
+	candidates []string,
+	destinations map[string]bool,
```

Comment:
Using `map[string]bool` here while `[]string` for `candiates` is awkward. Please use `[]string` and compute map internally if needed.

Reply:
Done